packages/spor-input-react/src/index.tsx

```diff
@@ -1,13 +1,6 @@
-export {
-  FormErrorMessage,
-  FormHelperText,
-  InputGroup,
-  InputLeftAddon,
-  InputLeftElement,
-  InputRightAddon,
-  InputRightElement,
```

Comment:
Note that we're removing the InputLeftAddon and InputRightAddon elements here. We don't yet have designs for those, but they will be added once we get that far.